src/containers/ProfileDetails/components/SSID/index.js

```diff
@@ -458,7 +482,10 @@ const SSIDForm = ({
           <Select
             data-testid="securityMode"
             className={globalStyles.field}
-            onChange={value => setMode(value)}
+            onChange={value => {
+              setMode(value);
+              handleOnModeChanged();
```

Comment:
I think setModeChanged(true) would have the same effect as making a function and doing a check